cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1574,6 +1574,15 @@
   </property>
 
   <property>
+    <name>cdap.ugi.cache.expiration.ms</name>
```

Comment:
Should we have different cache values for delegation token caching (in containers) vs Kerberos TGT caching (in master)?
These timeout values could be different.